src/env.ts

```diff
@@ -156,7 +156,7 @@ const envDefinitions = {
     desc: "location of the compound API",
   },
   COSMOS_GAS_AMPLIFIER: {
-    def: 1.4,
+    def: 4,
```

Comment:
Is it wanted to revert this value? Why?

Reply:
Fix in 92f4d599d10d0f6dad8c52c0035104a4a296159a.
This is a workarround for mismatched fees estimation calculation (simulation and broadcast are on different endpoints)